kong/tools/mime_type.lua

```diff
@@ -88,7 +92,44 @@ local function parse_mime_type(mime_type)
   return media_type:match(mime_type)
 end
 
+--- Checks if this mime-type includes other mime-type
+-- @tparam table this This mime-type
+-- @tparam table other Other mime-type
+-- @treturn boolean Returns `true` if this mime-type includes other, `false` otherwise
+local function includes(this, other)
+  if not other then
+    return false
+  end
```

Comment:
When would it be correct to pass `nil`,`false` or some other falsy value as `other`?  Does this create the potential of silently papering over bugs when the caller passes in something falsy?